src/Form/GCSettingsForm.php

```diff
@@ -80,7 +81,108 @@ public function buildForm(array $form, FormStateInterface $form_state): array {
       '#required' => TRUE,
     ];
 
-    $form['app_settings']['components'] = [
+    $form['app_settings']['switch_legacy_view'] = [
+      '#title' => $this->t('Switch to Legacy View'),
+      '#description' => $this->t('Legacy View has the following components: Virtual Y Video, Live streams, Virtual meetings, Blog posts.'),
```

Comment:
Legacy View has the following components `on the Home page`: Virtual Y Video, Live streams, Virtual meetings, Blog posts.